docs/concourse/README.md

```diff
@@ -32,6 +32,14 @@ To consume the project with our terragrunt code and scripts please create a fold
 * Use `git resource` for terraform modules: see [terragrunt/concourse-wg-ci-test/config.yaml](../../terragrunt/concourse-wg-ci-test/config.yaml) or
 copy `terraform-modules` folder to your repository, see [terragrunt/concourse-wg-ci/config.yaml](../../terragrunt/concourse-wg-ci/config.yaml)
 
+:warning: If you reference terraform modules with a tagged git revision, make sure to use the same tagged revision of `.tools-versions`. Otherwise, there will be version mismatch errors when you run terragrunt.
```

Comment:
```suggestion
:warning: If you reference terraform modules with a tagged git revision, make sure to use the same tagged revision of `.tools-versions`. Otherwise, there will be version mismatch errors when you run terragrunt. Alternatively, make use of the file `flake.nix` via `nix develop` or via direnv-load, see [direnv documentation](https://direnv.net/man/direnv-stdlib.1.html#codeuse-flake-ltinstallablegtcode)
```

Reply:
As the same effect could be mitigated by using the versions tagged by the `flake.lock`, I propose to provide that hint as well. We from autoscaler may abandon asdf soon.